components/prize_pool/commons/prize_pool_legacy.lua

```diff
@@ -55,7 +55,7 @@ function LegacyPrizePool.run(dependency)
 
 	newArgs.prizesummary = (header.prizeinfo and not header.noprize) and true or false
 	newArgs.cutafter = header.cutafter
-	newArgs.lpdb_prefix = header.lpdb_prefix or header.smw_prefix
+	newArgs.lpdb_prefix = header.lpdb_prefix
```

Comment:
would need bot run across all wikis
not sure what non git modules use that param and where it could potentially cause issues to just switch it

Reply:
Only Module:TeamCard/Custom, and only as a parameter name. Should be no issue

Reply:
the cross wiki search found >1.6k results for `insource:"smw_prefix"`

Reply:
I know